ecl/hql/hqlusage.cpp

```diff
@@ -344,6 +344,7 @@ static IPropertyTree * addSelect(IPropertyTree * xml, IHqlExpression * expr, boo
     Owned<IPropertyTree> field = createPTree(isUsed ? "field" : "unused");
     field->setProp("@name", text.str());
     const char * tag = field->queryName();
+    xml = ensurePTree(xml, "fields");
```

Comment:
This is another format change.  Is it required?  Is it due to the cassandra integration?